stable/application-mesh/tests/golden/components/egress.golden.yaml

```diff
@@ -0,0 +1,337 @@
+---
+# Source: application-mesh/templates/egress.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egress-leads-broker-db
+spec:
+  host: istio-egress-gateway.istio-ingress.svc.cluster.local
+  subsets:
+  - name: default
+---
+# Source: application-mesh/templates/egress.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egress-leads-broker-redis
+spec:
+  host: istio-egress-gateway.istio-ingress.svc.cluster.local
+  subsets:
+  - name: default
+---
+# Source: application-mesh/templates/egress.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egress-leads-broker-some-external-website
+spec:
+  host: istio-egress-gateway.istio-ingress.svc.cluster.local
+  subsets:
+  - name: default
+---
+# Source: application-mesh/templates/egress.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egress-kafka
+spec:
+  host: istio-egress-gateway.istio-ingress.svc.cluster.local
+  subsets:
+  - name: default
+---
+# Source: application-mesh/templates/egress.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: egress-leads-broker-db
+spec:
+  selector:
+    istio: egress-gateway
+  servers:
+    - port:
+        number: 443
+        name: tls
+        protocol: TLS
+      hosts:
+      - lead-broker-staging.aws.db
+      tls:
+        mode: PASSTHROUGH
+---
+# Source: application-mesh/templates/egress.yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: egress-leads-broker-redis
+spec:
+  selector:
+    istio: egress-gateway
+  servers:
+    - port:
+        number: 443
```

Comment:
Something is off on db/redis ports in Gateway/VirtualService resources. Note that kafka/http seems to be using correct poprts, while db and redis using 443